CONTRIBUTING.md

```diff
@@ -7,6 +7,39 @@ The Snap in this repository is specifically developed for the
 
 ## Developing and testing
 
+**Required software:**
+
+- [Snapcraft](https://snapcraft.io/docs/snapcraft-overview)
+- [Multipass](https://multipass.run/)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+**Building Magma AGW snap:**
+
+1. Install required software listed above
+2. From the repository's main directory execute:
+
+   ```bash
+   snapcraft
+   ```
+   
+   To see what's happening during Snap's building process, `-d` can be used along with above
+   command.
+
+**Installing locally built Magma AGW snap:**
+
+1. Copy snap to AGW host machine.
+2. On AGW host machine, as `root` execute:
+
+   ```bash
+   snap install <PATH_TO_THE_SNAP_FILE> --dangerous --classic
+   ```
+   Since Snap has been built locally, `--dangerous` flag is required to deploy it. <br>
```

Comment:
No need for explanations like `Since Snap ...`

Reply:
Removed.